app/models/ingredient_meal.rb

```diff
@@ -27,4 +31,21 @@ def price
   private def set_meal_non_veg
     meal.set_non_veg
   end
+
+  private def set_branch_meal_availability
+    if ingredient_quantity_previously_changed?
+      target_branches_meals = branches_meals.joins(:inventories).where('inventories.ingredient_id = ?', ingredient_id)
```

Comment:
call ingredient meal method

Reply:
if the request is taking too much time..We can add delayed jobs

Reply:
Did not make job for now. Updated